nrm_django/grants/admin.py

```diff
@@ -16,9 +16,9 @@ class GrantAuthorityInline(admin.TabularInline):
 class GrantAdmin(admin.ModelAdmin):
     inlines = [GrantAuthorityInline, NoteInline]
     list_display = (
-        "pretty_name",
         "gid",
         "pretty_cooperator_name",
+        "pretty_name",
```

Comment:
Same concern, changing `GrantAdmin` seems unrelated to the contacts changes?

Reply:
I was sneaking a small admin change in based on last week's feedback.